pkg/api/validate/admin.go

```diff
@@ -80,7 +79,7 @@ func (v *AdminAPIValidator) validateUpdateContainerService(cs, oldCs *api.OpenSh
 	old.Config.Images = cs.Config.Images
 
 	if !reflect.DeepEqual(cs, old) {
-		errs = append(errs, fmt.Errorf("invalid change %s", deep.Equal(cs, old)))
+		errs = append(errs, fmt.Errorf("invalid change %s", cmp.Diff(cs, old)))
```

Comment:
This and the following changes:

* https://github.com/openshift/openshift-azure/pull/1792/files#diff-9d1ebcff0c2c4390297fac3eb1aaebb1R82
* sync client v5: https://github.com/openshift/openshift-azure/pull/1792/files#diff-6b9eb417c2465f803e04c5286a756578R242
* same as above but in v6: https://github.com/openshift/openshift-azure/pull/1792/files#diff-323d45f40968c6b37fb8b3272de42e01R252

Change the logs output. But I didn't find any code/tools that depends on the output format. I assume it's safe to change. Please, correct me if I'm wrong.